Source/Core/Core/HW/WiiSaveCrypted.cpp

```diff
@@ -88,7 +88,7 @@ void CWiiSaveCrypted::ExportAllSaves()
 			}
 		}
 	}
-	SuccessAlertT("Found %u save files", (unsigned int)titles.size());
+	SuccessAlertT("Found %u save file%s", (unsigned int)titles.size(), (unsigned int)titles.size() != 1 ? "s" : "" );
```

Comment:
This is not going to properly translate, so in the end this is making it worse for every non-english language.